templates/add_page.html

```diff
@@ -167,6 +168,8 @@
             var claim_text = $('#claim_text').val();
             if (claim_text.length < AddPage.min_claim_text_len) {
                 $('#claim_text').addClass('incorrect');
+                $('#claim_text').val(AddPage.min_claim_text_len + " characters is minimum length of text");
+
```

Comment:
I think we should keep original value to let user modify it even if it incorrect so far. Massege can be shown in some div or popup.